images/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine:latest
+  
+RUN apk add --update git openssh-client
```

Comment:
Should this run `apk update` too?

Reply:
I believe the --update flag fetches the current package index before adding the package.